model_compression_toolkit/target_platform_capabilities/target_platform/targetplatform2framework/attach2keras.py

```diff
@@ -0,0 +1,106 @@
+            OperatorSetNames.OPSET_BATCH_NORM.value: [BatchNormalization],
+            OperatorSetNames.OPSET_RELU.value: [tf.nn.relu, ReLU],
+            OperatorSetNames.OPSET_RELU6.value: [tf.nn.relu6],
+            OperatorSetNames.OPSET_LEAKY_RELU.value: [tf.nn.leaky_relu, LeakyReLU],
+            OperatorSetNames.OPSET_HARD_TANH.value: [LayerFilterParams(Activation, activation="hard_tanh")],
+            OperatorSetNames.OPSET_ADD.value: [tf.add, Add],
+            OperatorSetNames.OPSET_SUB.value: [tf.subtract, Subtract],
+            OperatorSetNames.OPSET_MUL.value: [tf.math.multiply, Multiply],
+            OperatorSetNames.OPSET_DIV.value: [tf.math.divide, tf.math.truediv],
+            OperatorSetNames.OPSET_MIN_MAX.value: [tf.math.minimum, tf.math.maximum, Minimum, Maximum],
+            OperatorSetNames.OPSET_PRELU.value: [PReLU],
+            OperatorSetNames.OPSET_SWISH.value: [tf.nn.swish, LayerFilterParams(Activation, activation="swish")],
+            OperatorSetNames.OPSET_SIGMOID.value: [tf.nn.sigmoid, LayerFilterParams(Activation, activation="sigmoid")],
+            OperatorSetNames.OPSET_TANH.value: [tf.nn.tanh, LayerFilterParams(Activation, activation="tanh")],
+            OperatorSetNames.OPSET_GELU.value: [tf.nn.gelu, LayerFilterParams(Activation, activation="gelu")],
+            OperatorSetNames.OPSET_HARDSIGMOID.value: [tf.keras.activations.hard_sigmoid,
+                                                       LayerFilterParams(Activation, activation="hard_sigmoid")],
+            OperatorSetNames.OPSET_FLATTEN.value: [Flatten],
+            OperatorSetNames.OPSET_GET_ITEM.value: [tf.__operators__.getitem],
+            OperatorSetNames.OPSET_RESHAPE.value: [Reshape, tf.reshape],
+            OperatorSetNames.OPSET_PERMUTE.value: [Permute],
+            OperatorSetNames.OPSET_TRANSPOSE.value: [tf.transpose],
+            OperatorSetNames.OPSET_DROPOUT.value: [Dropout],
+            OperatorSetNames.OPSET_SPLIT.value: [tf.split],
+            OperatorSetNames.OPSET_MAXPOOL.value: [MaxPooling2D],
+            OperatorSetNames.OPSET_SHAPE.value: [tf.shape, tf.compat.v1.shape],
+            OperatorSetNames.OPSET_EQUAL.value: [tf.math.equal],
+            OperatorSetNames.OPSET_ARGMAX.value: [tf.math.argmax],
+            OperatorSetNames.OPSET_TOPK.value: [tf.nn.top_k],
+            OperatorSetNames.OPSET_FAKE_QUANT_WITH_MIN_MAX_VARS.value: [tf.quantization.fake_quant_with_min_max_vars],
+            OperatorSetNames.OPSET_COMBINED_NON_MAX_SUPPRESSION.value: [tf.image.combined_non_max_suppression],
+            OperatorSetNames.OPSET_CROPPING2D.value: [Cropping2D],
+            OperatorSetNames.OPSET_ZERO_PADDING2d.value: [ZeroPadding2D],
+            OperatorSetNames.OPSET_CAST.value: [tf.cast],
+            OperatorSetNames.OPSET_STRIDED_SLICE.value: [tf.strided_slice]
+        }
+
+        if FOUND_SONY_CUSTOM_LAYERS:
+            self._opset2layer[OperatorSetNames.OPSET_POST_PROCESS] = [SSDPostProcess]
```

Comment:
@lior-dikstein assuming this is the operator set name that is going to be added to the enum for pre process